csrc/multidevice/communicator.h

```diff
@@ -35,6 +35,8 @@ using RankType = DeviceIdxType;
 // Supported backends. TODO: gloo untested
 enum class CommunicatorBackend { nccl, ucc, gloo };
 
+std::ostream& operator<<(std::ostream& out, const CommunicatorBackend& cb);
```

Comment:
Missing implementation?

Reply:
implementation is already here https://github.com/NVIDIA/Fuser/blob/3434695576448f2dc5534deec967a60e295b27e6/csrc/multidevice/communicator.cpp#L30

It was just missing in the header